flake.nix

```diff
@@ -0,0 +1,66 @@
+{
+  description = "EvaP";
+
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+    poetry2nix = {
+      url = "github:nix-community/poetry2nix";
+      inputs.nixpkgs.follows = "nixpkgs";
+    };
+
+    flake-parts.url = "github:hercules-ci/flake-parts";
+    process-compose-flake.url = "github:Platonic-Systems/process-compose-flake";
+    services-flake.url = "github:juspay/services-flake";
```

Comment:
I don't think that updating nix itself can break anything. If anything, updating the commit of the dependency itself could break something, but this can only happen when we update `flake.lock`

Both process-compose-flake and services-flake have commits from a couple of days ago, but yeah, they could of course die at any time. If that happens, we can always write our own configuration file for process compose - this is really what these two dependencies do.